sdk/synapse/Azure.Analytics.Synapse.Artifacts/src/autorest.md

```diff
@@ -26,6 +26,32 @@ directive:
   transform: $.format = "url"
 ```
 
+### Add nullable annotations
```

Comment:
Are there corresponding swagger spec changes for these?

Reply:
Waiting to be merged: https://github.com/Azure/azure-rest-api-specs/pull/16237

Reply:
Tracking with https://github.com/Azure/azure-sdk-for-net/issues/24399